react-native-adjust.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name           = 'react-native-adjust'
+  s.version        = package['version']
+  s.summary        = package['description']
+  s.description    = package['description']
+  s.license        = package['license']
+  s.author         = package['author']
+  s.homepage       = package['homepage']
+  s.source         = { :git => 'https://github.com/adjust/react_native_sdk', :tag => s.version }
+
+  s.requires_arc   = true
+  s.platform       = :ios, '8.0'
+
+  s.preserve_paths = 'LICENSE', 'README.md', 'package.json', 'index.js'
+  s.source_files   = 'ios/*.{h,m}'
```

Comment:
@uerceg with the way we're using it, you wouldn't need to publish the pod directly, but rather just add the podspec to this project. We currently do this on our project with other libraries, for example:

https://github.com/rebeccahughes/react-native-device-info/blob/12c93cbe36af2489d5710f0d300ae4c08622cf7c/RNDeviceInfo.podspec

https://github.com/davodesign84/react-native-mixpanel/blob/0e32006a1184f82d498b2bd0fdd7d3241ced8c5b/react-native-mixpanel.podspec

https://github.com/oblador/react-native-keychain/blob/d3282cdbca0bdadff83c32061af43d2c2f63c166/RNKeychain.podspec

And then in our `Podfile` we have:

```
pod 'RNDeviceInfo', :path => './node_modules/react-native-device-info'
pod 'RNKeychain', :path => './node_modules/react-native-keychain'
pod 'react-native-mixpanel', :path => './node_modules/react-native-mixpanel'
```

If you add the podspec with the recursive glob we'll be able to add:

```
pod 'react-native-adjust', :path => './node_modules/react-native-adjust'
``